Gems/ROS2/Code/Source/ImGui/SensorImGui.h

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#include <ROS2/ImGui/ImGuiBase.h>
+
+#include "../Camera/ROS2CameraSensorComponent.h"
+#include "../ContactSensor/ROS2ContactSensorComponent.h"
+#include "../GNSS/ROS2GNSSSensorComponent.h"
+#include "../Imu/ROS2ImuSensorComponent.h"
+#include "../Lidar/ROS2LidarSensorComponent.h"
+#include "../Odometry/ROS2OdometrySensorComponent.h"
+
+#pragma once
+
+namespace ROS2
+{
+    class SensorImGui
```

Comment:
There is no documentation. I do know what this class is for.

Reply:
Moved the file.